### PR TITLE
show dif page for dotnet

### PR DESCRIPTION
**Pull request by bruno-garcia**
### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

**Review by mattjohnsonpint — APPROVED** [review]

**Comment by mattjohnsonpint** [comment]
Actually, maybe we should wait until we resolve this?

image

**Comment by bruno-garcia** [comment]
> Actually, maybe we should wait until we resolve this?
> 
> 
> 
> image
> 
> 

This documentation clarifies debug formats we rely on now for mobile. So I think we shouldn't wait.

Might be worth a Note on the top when rendering under dotnet. Saying it's just mobile right now. I'll add.

File: src/platforms/common/data-management/debug-files/index.mdx [hunk 1]
```diff
@@ -9,6 +9,8 @@ supported:
   - cocoa
   - apple
   - unity
+  - dotnet
+  - dotnet.maui
```
Comment (bruno-garcia) [inline src/platforms/common/data-management/debug-files/index.mdx:13]:
```suggestion
```

**Comment by bruno-garcia** [comment]
Adding the page for now. It's just a general Debug Info documentation, not built to be specific to what exactly each platform can leverage (maybe just `native` other than ProGuard stuff).

I'll raise an issue to refactor this section to better include/exclude each debug info type where it makes sense to be available

**Pull request merged**